tiny-event-sourcing-lib/src/main/kotlin/ru/quipy/streams/EventStoreReader.kt

```diff
@@ -100,12 +99,15 @@ class EventStoreReader(
     }
 
     private fun launchEventStoreReaderHealthCheckJob(): Job {
-        return CoroutineScope(CoroutineName("reading-$streamName-coroutine") + dispatcher).launch {
+        return CoroutineScope(CoroutineName("reading-$streamName-event-store-coroutine") + dispatcher).launch {
             while (isHealthcheckActive.get()) {
                 if (streamManager.hasActiveReader(streamName)) {
```

Comment:
what if `streamManager.hasActiveReader(streamName)) == true` but the current instance is "active" one. So till the healthCheck period is exipired the current instance does not update the "last interaction"?